Core/src/main/java/com/plotsquared/core/util/helpmenu/HelpPage.java

```diff
@@ -68,7 +68,7 @@ public void render(PlotPlayer<?> player) {
                             )
                     )
             );
-            Placeholder<?> help_objects = Placeholder.miniMessage("help_objects", StringMan.join(this.helpObjects, "\n"));
```

Comment:
The help menu looks even more cluttered now
![](https://i.imgur.com/NJmtiZh.png)
We can safely get a rid of all line breaks.

Reply:
Removed the other line breaks as well